tgbot/config.py

```diff
@@ -13,33 +14,37 @@ class DbConfig:
 @dataclass
 class TgBot:
     token: str
-    admin_id: int
+    admin_ids: int
     use_redis: bool
 
 
+@dataclass
+class Miscellaneous:
+    other_params: str = None
+
+
 @dataclass
 class Config:
     tg_bot: TgBot
     db: DbConfig
+    misc: Miscellaneous
 
 
-def cast_bool(value: str) -> bool:
-    if not value:
-        return False
-    return value.lower() in ("true", "t", "1", "yes")
-
-
-def load_config(path: str):
-    config = configparser.ConfigParser()
-    config.read(path)
-
-    tg_bot = config["tg_bot"]
+def load_config(path: str = None):
+    env = Env()
+    env.read_env(path)
```

Comment:
Are you sure, that docker-compose and Environs use the same format of `.env` file?
I'd rather split this cases if we really need both.